python/graphstorm/run/gsgnn_np/gsgnn_np.py

```diff
@@ -86,7 +86,10 @@ def main(config_args):
     if trainer.rank == 0:
         tracker.log_params(config.__dict__)
     trainer.setup_task_tracker(tracker)
-    device = 'cuda:%d' % trainer.dev_id
+    if th.cuda.is_available():
```

Comment:
We can have a setup_device() function in graphstorm.utils.py as:

```
def setup_device(local_rank):
    """ Setup device

    Parameters
    -----------
    local_rank: int
        Locak rank, same as gpu id.
    """
    if th.cuda.is_available():
        device = 'cuda:%d' % local_rank
        th.cuda.set_device(device)
    else:
        device="cpu"

    return device
```

Reply:
Here is an example: https://github.com/awslabs/graphstorm/pull/337/files#diff-fb641736e08124342fd53ef3ba61dd132f6e25851fe261250d536573a23fc8e7

Reply:
I can change it like that if you prefer so. But do we want to change it here if it is already done in another incoming PR?

Reply:
Let's do it in this PR. This can make codes simpler.
That PR is a draft.